d2bs/kolbot/libs/scripts/Follower.js

```diff
@@ -177,6 +178,15 @@ const Follower = new Runnable(
             announce("Failed to enter red cow portal.");
           }
         }));
+      ["ubers", (me.name + " ubers")]
```

Comment:
This can be removed in favor of adding

```js
// Uber portals
      const uberPortals = [
        sdk.areas.MatronsDen, sdk.areas.ForgottenSands, sdk.areas.FurnaceofPain, sdk.areas.UberTristram
      ];
      if (me.inArea(sdk.areas.Harrogath) && uberPortals.includes(area)) {
        Town.moveToSpot("stash");
        let uberPortal = Pather.getPortal(area);

        if (uberPortal) {
          announce("Special transit to " + getAreaName(area));
          return Pather.usePortal(null, null, uberPortal);
        }
      }
  ```
  
  into checkExit

Reply:
sorry for my delayed reaction. was quite stressful time recently